### ISSUE TITLE
Despite WITH_CERES option, ceres is actually required to compile

### ISSUE THREAD

**Issue by severin-lemaignan** [strands-project/v4r#45]
While the CMakeLists exposes a WITH_CERES option, ceres is indeed required to compile v4r, due to `modules/reconstruction/include/v4r/reconstruction/impl/ReprojectionError.hpp` including `ceres/rotation.h`.

**Comment by ToMadoRe** [comment]
Yes, you are right. At the moment, we do not conditionally compile files / modules depending on whether these 3rd party libraries are installed. This would require to rewrite some cmake files of V4R but is planned to be implemented in the future.

**Comment by CWallbridge** [comment]
I seem to have encountered a similar issue. Also the libceres-dev package that can be installed with Ubuntu does not appear to provide the CeresConfig.cmake or ceres-config.cmake file required for being built WITH_CERES enabled. Is there somewhere I can get these files?

**Comment by taketwo** [comment]
I would recommend to compile *Ceres* from source and tell *CMake* where you installed it with `-DCeres_DIR=<install_dir>/share/Ceres`.
